### ISSUE TITLE
infestation should really count towards beelzebub

### ISSUE THREAD

**Issue by BadsyEdgecat** [UnderscoreKilburn/repentance-issues#1471]
its an onhit effect that spawns flies, and its a maggot. it seems weird that it hasnt done this in the past

**Comment by ghost** [comment]
Agreed, i feel like Infestation is the most underwhelming item out of the passive items that don't actively hurt your run. If it was part of Beelzebub and maybe spawned some flies on pickup too it would be a bit more interesting at least.

**Comment by Rchardon** [comment]
Added in Rep+
